autoInstaller.sh

```diff
@@ -420,6 +423,10 @@ if (($installgtm || $installYottaDB) && ! $generateViVDox); then
       mv $dir VistA-Source
   fi
 
+  echo "Install dependencies for GT.M/YDB Kernel Utilities"
```

Comment:
why not put all of this in the main yum install? There is only one package (bind-utils) that isn't shared between both.